module/zfs/spa.c

```diff
@@ -3100,6 +3218,21 @@ spa_load_impl(spa_t *spa, uint64_t pool_guid, nvlist_t *config,
 		}
 	}
 
+	if (spa_feature_is_active(spa, SPA_FEATURE_ALLOCATION_CLASSES)) {
+		uint64_t class_stats_obj;
+
+		if (spa_dir_prop(spa, DMU_POOL_NORMAL_CLASS_STATS,
+		    &class_stats_obj) == 0) {
```

Comment:
how is it possible for the feature to be active, but the stats don't exist?

Reply:
this is a load; renamed to avoid confusion